solution/backend/regulations/views/reader.py

```diff
@@ -56,6 +57,15 @@ def get_context_data(self, **kwargs):
         for location in locations:
             resource_count[location.display_name] = location.num_locations
 
+        conversions = {}
+        for section, usc, act, title in StatuteLinkConverter.objects.values_list("section", "usc", "act", "title"):
+            if act not in conversions:
+                conversions[act] = {}
+            conversions[act][section] = {
```

Comment:
Section is not a required field of the model so what will happen if there is no section?

Reply:
Resolved in pairing discussion